### PR TITLE
arm-forge: add 22.1.2

### PR DESCRIPTION
**Pull request by louisespellacy-arm**

### CODE REVIEW

File: var/spack/repos/builtin/packages/arm-forge/package.py [hunk 1]
```diff
@@ -99,6 +105,9 @@ class ArmForge(Package):
         )
         version("21.0", sha256="60cfa7dd1cd131ec85e67cb660f2f84cf30bb700d8979cae1f5f88af658fd249")
     elif platform.machine() == "x86_64":
+        version(
+            "22.1.2", sha256="baeac2b49c38547a8f3dd6a5237544eefe8c7695d969784e6f861ce77b610542"
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/arm-forge/package.py:109]: Confirmed this sha256.

**Review by tldahlgren — APPROVED** [review]
Confirmed the sha256 for one of the platforms.

Will give @NickRF time to comment.

**Review by bernhardkaindl — APPROVED** [review]
22.1.2 built fine on Linux-arm64, so I think this looks good.

**Review by NickRF — APPROVED** [review]
Confirmed sums

**Comment by louisespellacy-arm** [comment]
Can these changes be merged now?

**Pull request merged**